search/search-domain/src/main/java/io/camunda/search/entities/AuditLogEntity.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.search.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public record AuditLogEntity(
+    Long auditLogKey,
+    String entityKey,
+    AuditLogEntityType entityType,
+    AuditLogOperationType operationType,
+    Long batchOperationKey,
+    BatchOperationType batchOperationType,
+    String timestamp,
+    String actorId,
+    AuditLogActorType actorType,
+    String tenantId,
+    AuditLogResult result,
+    String annotation,
+    AuditLogCategory category,
+    String processDefinitionId,
+    Long processDefinitionKey,
+    Long processInstanceKey,
+    Long elementInstanceKey,
```

Comment:
🔧 Let's include `deploymentKey`, `formKey`, and `resourceKey` here. We need to provide them for resource operations, the scope was adjusted ([src](https://docs.google.com/document/d/1COzShIcqFqZlq6ZRfuL9VeHD7oNvVd8vhnezxGlYTU0/edit?tab=t.c482jdb4ehw2#heading=h.290rkeyts93t)).